libs/payments/ui/src/lib/server/terms-and-privacy.ftl

```diff
@@ -0,0 +1,5 @@
+## Component - TermsAndPrivacy
+
+next-terms = Terms of Service
+next-privacy = Privacy Notice
+next-terms-download = Download Terms
```

Comment:
@bcolsson these are some examples of strings that are also used in SP2.5 - fxa-payments-server. To ensure we aren't duplicating, we'll prefix these with `next-`. I hope that works?